src/components/ui/index.js

```diff
@@ -0,0 +1,3 @@
+export { CoralIconButton } from './CoralIconButton';
+export { CoralIcon } from './CoralIcon';
+export { CoralButton } from './CoralButton';
```

Comment:
please set up your text editor to automatically add a newline to the end of the file upon save. I think all the files in the repo are this way, so we're going to get unnecessary diffs otherwise.